oscillogram_classification/cluster.py

```diff
@@ -182,7 +212,14 @@ def create_processed_time_series_dataset(data_path: str, norm: bool) -> None:
     """
     voltage_series = []
     labels = []
-    for path in Path(data_path).glob('**/*.csv'):
+    measurement_ids = []
+
+    if os.path.isfile(data_path):
+        paths = list(Path(data_path))
```

Comment:
`paths = [Path(data_path)]` instead of `list()` - removes the warning, `list` expects an `iterable`